src/tools/qt-gui/src/confignode.hpp

```diff
@@ -252,6 +257,7 @@ class ConfigNode : public QObject
 
 	bool m_isExpanded;
 	bool m_isDirty;
+	bool m_root;
```

Comment:
I wouldn't call these 'm_root' but 'm_namespace' instead (because there is only one single root node).

Reply:
The name change includes the methods as well of course